store/demo/dev/10221__sheet_organizer.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO
```

Comment:
Let's also create more sheet seed data, otherwise , the sheet view and bookmark view looks the same.